micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmGcMetrics.java

```diff
@@ -170,17 +170,19 @@ public void handleNotification(Notification notification, Object ref) {
             CompositeData cd = (CompositeData) notification.getUserData();
             GarbageCollectionNotificationInfo notificationInfo = GarbageCollectionNotificationInfo.from(cd);
 
+            String gcName = notificationInfo.getGcName();
             String gcCause = notificationInfo.getGcCause();
             String gcAction = notificationInfo.getGcAction();
             GcInfo gcInfo = notificationInfo.getGcInfo();
             long duration = gcInfo.getDuration();
-            if (isConcurrentPhase(gcCause, notificationInfo.getGcName())) {
-                Timer.builder("jvm.gc.concurrent.phase.time").tags(tags).tags("action", gcAction, "cause", gcCause)
+            if (isConcurrentPhase(gcCause, gcName)) {
+                Timer.builder("jvm.gc.concurrent.phase.time").tags(tags)
+                        .tags("action", gcAction, "cause", gcCause, "gc_name", gcName)
```

Comment:
This is inconsistent to `gc` in `jvm.gc.pause` and I'm thinking if this should be a separate PR.

Reply:
I see. How dumb of me to have contrasting names in consecutive conditional blocks 😟. I have no issues having this part of a separate PR.